MADRaS/traffic/const_vel.py

```diff
@@ -2,17 +2,17 @@
 import sys
```

Comment:
Also the code was not able to find the imports of lines 
```python
from MADRaS.controllers.pid import PID
from MADRaS.utils.gym_torcs import TorcsEnv
import MADRaS.utils.snakeoil3_gym as snakeoil3
from MADRaS.utils.madras_datatypes import Madras
```

so I had to add the line `sys.path.append("/home/sohan/projects/MADRaS/MADRaS")` for it to work.

Reply:
How did you test the traffic agents ?

Reply:
I did it using the same way as you suggested. 
Upon removing the line this is the error traceback that I get.
```shell
  File "/home/sohan/anaconda2/envs/madras/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2338, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/sohan/projects/MADRaS/MADRaS/envs/__init__.py", line 2, in <module>
    from MADRaS.envs.gym_madras import MadrasEnv
  File "/home/sohan/projects/MADRaS/MADRaS/envs/gym_madras.py", line 22, in <module>
    import MADRaS.traffic.vel_change as agentSinusoid
  File "/home/sohan/projects/MADRaS/MADRaS/traffic/vel_change.py", line 6, in <module>
    from controllers.pid import PID
ImportError: No module named 'controllers'

```

Reply:
Oh sorry,  got the problem

Reply:
Actually the other traffic files do noyt have their imports correctly set up. [vel_change.py](MADRaS/traffic/vel_change.py)